config/initializers/airbrake.rb

```diff
@@ -1 +1,6 @@
-# This is overwritten on deploy
+Airbrake.configure do |config|
+  config.api_key = ENV['ERRBIT_API_KEY']
+  config.host = "errbit.#{ENV['GOVUK_APP_DOMAIN']}"
```

Comment:
It would be nice if this was something like ENV['ERRBIT_HOST'] or "errbit.#{ENV['GOVUK_APP_DOMAIN']}" such that it is fully configurable.

(same as https://github.com/alphagov/panopticon/pull/412#discussion_r88450810 )